cleanlab/datalab/issue_manager/duplicate.py

```diff
@@ -125,7 +125,7 @@ def _neighbors_within_radius(knn_graph: csr_matrix, radius: float):
         N = knn_graph.shape[0]
         distances = knn_graph.data.reshape(N, -1)
         # Create a mask for the threshold
-        mask = distances < radius
```

Comment:
can you confirm the set of examples flagged as near duplicates has not changed, i.e. this PR only updates the sets of near_duplicates in the info object?

Reply:
The `issues` dataframe that flags near duplicates is set only once in the `find_issues` method. This occurs before the call to the `_neighbors_within_radius` which computes the sets of near-duplicates (which ends up in the info object).

I can confirm: The flagged examples have **not changed**.